.gitignore

```diff
@@ -25,10 +25,6 @@ var/
 .installed.cfg
 *.egg
 
-#Symlink
-normandy
-delivery-console
-
```

Comment:
Oh and I had to get rid of this otherwise I coudln't commit new files to the `normandy` app, so we'll have to figure out a better solution to this.  Maybe using different names for the symlinks that are unique and explicit like `normandy-app` or something?